javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -63,15 +63,30 @@ public void testRunningBazelInGivenRelativeDirectory() throws Exception {
 
   @Test
   public void testTestSuiteExists() throws Exception {
-    loadIntegrationTestRuleIntoWorkspace();
-    setupPassingTest("IntegrationTestSuiteTest");
+    setUpTestSuit("IntegrationTestSuiteTest");
 
     Command cmd = driver.bazel("test", "//:IntegrationTestSuiteTest");
     final int exitCode = cmd.run();
 
     org.hamcrest.MatcherAssert.assertThat(exitCode, is(successfulExitCode(cmd)));
   }
 
+  @Test
+  public void testUseBazelRcFile() throws Exception {
+    setUpTestSuit("IntegrationTestSuiteTest");
```

Comment:
Why do we need here loadIntegrationTestRuleIntoWorkspace? I’d rather test the Driver functionality in the DriverTest

Reply:
To test the driver I'll need to expose the `Command` internals (`args`)

Reply:
Why do you think so? 
What I meant is exactly this test BUT instead of having the test you’re running be bazel_java_integration_testing you can have it be java_test.
You still run bazel and everything but on a much simpler case.
This test is complicated since it’s a meta test.
Your test AFAIU is a simpler one.
Does that make sense?